postgresql/_database.sls

```diff
@@ -39,6 +39,21 @@ postgresql_database_{{ svr_name|default('localhost') }}_{{ database_name }}:
     - user: root
     {%- endif %}
 
+{%- for extension in database.get('extensions', []) %}
+postgresql_database_{{ svr_name|default('localhost') }}_{{ database_name }}_{{ extension }}:
+  postgres_extension.present:
+    - name: {{ extension }}
+    - maintenance_db: {{ database_name }}
+    - schema: public
+    - user: postgres
```

Comment:
User should be root as for other resources above, because postgres user may not be present on system where client is running.